src/triggers/Status_change1.trigger

```diff
@@ -0,0 +1,3 @@
+trigger Status_change1 on Timesheet__c (before insert) {
+	  System.debug('Hello World!');
+}
```

Comment:
You shoud have only one trigger per object, which then holds all the required  events.